Resources/config/services.yml

```diff
@@ -23,5 +24,4 @@ services:
             $translator: '@?translator'
             $templating: '@?templating.engine.twig'
 
-    Chaplean\Bundle\ApiClientBundle\Command\ChapleanApiLogCleanCommand:
-        autoconfigure: true
+    Chaplean\Bundle\ApiClientBundle\Command\ChapleanApiLogCleanCommand: ~
```

Comment:
Are you sure it is necessary? Should work without it.

Reply:
It would not register the command otherwise. So it is needed.

Reply:
Disagree! https://symfony.com/doc/current/console.html#registering-the-command

> Symfony commands must be registered as services and tagged with the console.command tag. If you're using the default services.yaml configuration, this is already done for you, thanks to autoconfiguration.